source/guide.html.haml

```diff
@@ -29,7 +29,11 @@ title: Sass Basics
 
       Once you start tinkering with Sass, it will take your preprocessed Sass file and save it out as a normal CSS file that you can use in your web&nbsp;site.
 
-      * * *
+      The most direct to make this happen is in your terminal. Once Sass is installed, you can run `sass input.scss output.css` from your terminal. You can watch either individual files or entire directories. In addition, you can watch folders or directories with the `--watch` flag. An example of running Sass while watching an entire directory is the following:
```

Comment:
You can ignore the houndci thingy. I should disable that.

In the meantime, I think a word is missing in the first sentence. "The most direct… way?"

Thanks so much for doing this btw!